configs/presets/devnet.yaml

```diff
@@ -30,5 +30,5 @@ bouncer_config:
     message_segment_length: 18446744073709551615
     n_events: 18446744073709551615
     state_diff_size: 131072
-sequencer_address: "0x123"
+sequencer_address: "0x0"
```

Comment:
Sequencer address cannot be zero for block production. We should use another one for the devnet config

Reply:
done!